CHANGELOG.md

```diff
@@ -30,6 +30,8 @@ More info [here](./docs/developers_guide.md#MinIO-S3-service)
 Other
 - vHive now also supports vanilla Knative benchmarking and testing (i.e., using containers for function sandboxes).
 More info [here](./docs/developers_guide.md#Testing-stock-Knative-images).
+- Added [script](./scripts/cloudlab/start_onenode_vhive_cluster.sh) to start vHive single node cluster.
```

Comment:
this should go to [Unrelease] above instead of v1.2

Reply:
moved